tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -0,0 +1,763 @@
+{
+    "$schema": "https://raw.githubusercontent.com/todogroup/repolinter/master/rulesets/schema.json",
+    "version": 2,
+    "axioms": {
+      "linguist": "language",
+      "licensee": "license",
+      "packagers": "packager"
+    },
+    "rules": {
+      "license-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}LICENSE*"
+            ],
+            "nocase": true
+          }
+        }
+      },
+      "security-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}SECURITY.md"
+            ]
+          }
+        }
+      },
+      "readme-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "README.md"
+            ],
+            "nocase": true
+          }
+        }
+      },
+      "contributing-file-exists": {
+        "level": "error",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}CONTRIBUTING.md"
+            ],
+            "nocase": true
+          }
+        }
+      },
+      "maintainers-file-exists": {
+        "level": "off",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}MAINTAINERS.md"
+            ]
+          }
+        }
+      },
+      "codeowners-file-exists": {
+        "level": "off",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}CODEOWNERS.md"
+            ]
+          }
+        }
+      },
+      "governance-file-exists": {
+        "level": "off",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}GOVERNANCE.md"
+            ]
+          }
+        }
+      },
+      "community-guidelines-file-exists": {
+        "level": "off",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}COMMUNITY_GUIDELINES.md"
+            ]
+          }
+        }
+      },
+      "code-of-conduct-file-exists": {
+        "level": "off",
+        "rule": {
+          "type": "file-existence",
+          "options": {
+            "globsAny": [
+              "{docs/,.github/,}CODE_OF_CONDUCT.md"
+            ]
+          }
+        }
+      },
+      "license-contains-license": {
+        "level": "error",
+        "rule": {
+          "type": "file-contents",
+          "options": {
+            "globsAll": [
+                "{docs/,.github/,}LICENSE*"
+            ],
+            "content": "license",
```

Comment:
@decause-gov Maybe we can check for specific licenses here like CC0?

Reply:
Since our templates will also be used by contractors in addition to career civil servants, I reckon it is acceptable to generally check for LICENSE existence. Perhaps in the warning message we can point to the CMS TRA/Open Source Policy, or generally the OSI approved license list: https://opensource.org/licenses or https://github.com/CMSgov/cms-open-source-policy/blob/main/Policy.md#license

Reply:
Looks like Repolinter doesn't support custom messages in the output. Guess we can find a way around.